pykubectl/objects.py

```diff
@@ -143,3 +176,28 @@ def execute(self, attempts=30):
     def logs(self, *args, **kwargs):
         cmd = f"logs {self.name}"
         return self.kubectl.execute(cmd, *args, **kwargs)
+
+class Job(KubeObject):
```

Comment:
I changed Job back to extend from KubeObject because we're no longer able to use either of the methods from Pod

Reply:
Ah right, Jobs use Pods to run conceptually but they have different API/CLI calls